custom_functions/url_parse.py

```diff
@@ -6,12 +6,13 @@ def url_parse(url=None, **kwargs):
         url (CEF type: url): The URL to parse
     
     Returns a JSON-serializable object that implements the configured data paths:
-        scheme: The scheme of the URL, such as HTTP, HTTPS, or FTP.
-        netloc: The network location of the URL, which is typically the hostname.
-        path: The path to the resource after the first slash in the URL, such as "en_us/software/splunk-security-orchestration-and-automation.html".
-        params: The parameters in the URL after the semicolon.
-        query: The query string of the URL after the question mark. Multiple parameters are not separated from each other.
-        fragment: The subcomponent of the resource which is identified after the hash sign.
+        scheme (CEF type: *): The scheme of the URL, such as HTTP, HTTPS, or FTP.
+        netloc (CEF type: domain): The network location of the URL, which is typically the hostname.
+        path (CEF type: *): The path to the resource after the first slash in the URL, such as "en_us/software/splunk-security-orchestration-and-automation.html".
+        params (CEF type: *): The parameters in the URL after the semicolon.
+        query (CEF type: *): The query string of the URL after the question mark. Multiple parameters are not separated from each other.
+        fragment (CEF type: *): The subcomponent of the resource which is identified after the hash sign.
+        originalUrl (CEF type: url): The original url that was parsed
```

Comment:
I think this should be input_url to be consistent with above. Also, editing it outside of Phantom made it so its currently giving a warning saying "Function definition may be invalid" when I import it.

Reply:
I went back and made the code changes inside of Phantom. I also changed url to input_url and made .netloc to be cef data type "domain." Commit: 
https://github.com/phantomcyber/playbooks/pull/17/commits/b30af756d08b3e64a227b556b737f79891d52490